No namespace (#21)
NLESC-JCER/run-cpp-on-web: js-webapp/README.md

> #ifndef H_NEWTONRAPHSON_H

Reviewer: ```suggestion
#ifndef H_NEWTONRAPHSON_HPP
```

Author: Can H_ be removed? Seems duplication to me
```suggestion
#ifndef NEWTONRAPHSON_HPP
```

Author: will do later

> #define H_NEWTONRAPHSON_H

Reviewer: ```suggestion
#define H_NEWTONRAPHSON_HPP
```

```html
<html>
   <head>
      <!-- Load WebAssemlby module -->
      <script type="text/javascript" src="newtonraphson.js"></script>
   </head>
   <body>
      <div>
         Function root is approximately at x =
         <span id="answer"/>
      </div>
      <script>
         // Wait for module to initialize,
         createModule().then(({NewtonRaphson}) => {
            // Hardcoded input values
            const tolerance = 0.001;
            const initial_guess = -20;
            // Perform computation
            const newtonraphson = new NewtonRaphson(tolerance);
            const root = newtonraphson.solve(initial_guess);
            // Write root to tag with answer as identifier

>             document.getElementById("answer").innerHTML = root.toFixed(2);

Reviewer: ```suggestion
            document.getElementById("answer").innerHTML = "Function root is approximately at x = " + root.toFixed(2);
```

Author: I moved the `Function root is approximately at x = ` to a html tag. So this line would be smaller.

Reviewer: just saw it.

         });
      </script>
   </body>
</html>
```
File: _index.html_.

### Hosting the app with a web server

> We can not just open this HTML page in a web browser, as the embedded JavaScript file can only be loaded when it is

Reviewer: ```suggestion
In order to display the HTML page in a web browser we will need a web server because the embedded JavaScript file can only be loaded in this way. We will use Python [http.server](https://docs.python.org/3/library/http.server.html) module for this.
```
